lib/vanagon/platform/deb.rb

```diff
@@ -99,7 +99,7 @@ def add_repository(definition, gpg_key = nil)
         # parse the definition and gpg_key if set to ensure they are both valid URIs
         definition = URI.parse(definition)
         gpg_key = URI.parse(gpg_key) if gpg_key
-        provisioning = ["apt-get -qq update && apt-get -qq install curl"]
+        provisioning = ["apt-get -qq update && DEBIAN_FRONTEND=noninteractive apt-get -qq install curl"]
```

Comment:
Not sure if we should just `export DEBIAN_FRONTEND=noninteractive` to have it everywhere or just fix it for curl install.

Reply:
I haven't spent much time mucking with DEBIAN_FRONTEND=noninteractive, and whatever weird side-effects it may bring,  but at least conceptually, I like what @gimmyxd is suggesting.

Reply:
I verified this fixes the issue for debian 18.04 due to curl pulling in a new libssl, which causes sshd to want to restart.